src/coreclr/src/jit/CMakeLists.txt

```diff
@@ -360,31 +360,23 @@ function(add_jit jitName)
     if(WIN32)
         add_definitions(-DFX_VER_INTERNALNAME_STR=${jitName}.dll)
     endif(WIN32)
-
     set_source_files_properties(${JIT_EXPORTS_FILE} PROPERTIES GENERATED TRUE)
-
     add_library_clr(${jitName}
         SHARED
         ${SHARED_LIB_SOURCES}
         ${JIT_ARCH_SOURCES}
     )
-
     target_precompile_header(TARGET ${jitName} HEADER jitpch.h ADDITIONAL_INCLUDE_DIRECTORIES ${JIT_SOURCE_DIR})
-
     add_dependencies(${jitName} jit_exports)
-
     set_property(TARGET ${jitName} APPEND_STRING PROPERTY LINK_FLAGS ${JIT_EXPORTS_LINKER_OPTION})
     set_property(TARGET ${jitName} APPEND_STRING PROPERTY LINK_DEPENDS ${JIT_EXPORTS_FILE})
-
     target_link_libraries(${jitName}
         ${JIT_LINK_LIBRARIES}
         ${JIT_ARCH_LINK_LIBRARIES}
     )
-
     # add the install targets
     install_clr(TARGETS ${jitName})
 endfunction()
-
```

Comment:
hmmm... you don't like whitespace?

Reply:
Didn't mean to do that. Let me undo that part.